app/templates/main/download-confirmation.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{%- from "govuk_frontend_jinja/components/accordion/macro.html" import govukAccordion -%}
+{%- from 'govuk_frontend_jinja/components/checkboxes/macro.html' import govukCheckboxes -%}
+{%- from 'govuk_frontend_jinja/components/select/macro.html' import govukSelect -%}
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from 'govuk_frontend_jinja/components/label/macro.html' import govukLabel -%}
+{%- from "outcomeCheckboxes.html" import outcomeCheckboxItems -%}
+{%- from "checkboxes.html" import checkboxItems -%}
+{%- from "select.html" import selectItems -%}
+
+{% block beforeContent %}
+{{ super() }}
+{% endblock %}
```

Comment:
With this implementation, this block looks redundant - can we just drop it altogether?